### DISCUSSION TITLE
feeds/memories endpoint locations

### DISCUSSION THREAD

**Discussion by dxrk** [userbradley/BeReal#26] [General]
Around this time last year, I tried to use the feeds/memories endpoint to make a map of all the locations I've taken my BeReal in the past year, and even though the coordinates are saved, it wasn't possible. I just decided to give it a shot again, and it's still not possible.

BeReal, for whatever reason, randomizes the coordinates when they are stored at this endpoint. It makes me wonder if it's intentional, so past locations can't be found, or if this is actually an error on their end. Wanted to see if you had noticed this before, or if I was doing something wrong. Just thought this was interesting and wanted to get another perspective!

**Comment by NeedNot** [comment]
i didn't look to see if they were "precise" but the memories seem to show the correct place for me. however some of my posts don't have locations while others do even though they did have location at the time
